mlir/include/mlir/Dialect/ArmSME/Transforms/Passes.td

```diff
@@ -122,4 +122,43 @@ def TileAllocation
   let dependentDialects = ["func::FuncDialect"];
 }
 
+def OuterProductWidening
```

Comment:
Hm, the pass doesn't "widen" the outer products, it "fuses"/"combines" them into one. WDYT?

Reply:
see what you mean, suppose could rename to `OuterProductFusion`?

Reply:
Yeah, or `SMEOuterProductFusion`.

Reply:
+1 to any name with fuse or fusion (sounds cool, and more descriptive :smile:)